### DISCUSSION TITLE
Directus on DreamHost

### DISCUSSION THREAD

**Discussion by adnanf2** [directus/directus#13604] [Archive]
I was able to setup Directus on my server using the CLI install but with a few issues;

1. I have to use Putty to keep the SSH session alive, if not then Directus stops working once the SSH session ends.
2. I am unable to access Directus using the subdomain I installed it on.
3. I am unable to access it using port 443, even though a Let's Encrypt SSL is assigned to that subdomain.

I was hoping to get some help in regards to accessing the admin panel via the sub-domain and enable SSL.
Currently I have to access Directus using the IP address of my server and port 8055.

I am running the latest version of Directus and using SQLite for the DB.
I was hoping someone can help me setup Directus properly on DreamHost.

**Comment by alexvdvalk** [comment]
If you are still interested in using Directus, you should consider using the Directus Cloud as we manage all this stuff for you.
